source/features/raw-view-link.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+const createRawUrl = (href: string): string => {
+	const url = href.split('/');
+	url[3] = 'raw'; // Replaces 'blob'
+	return url.join('/');
+};
+
+const observer = new MutationObserver(([{addedNodes}]) => {
+	for (const node of addedNodes) {
+		if (node instanceof Element) {
+			if (node.matches('js-diff-progressive-container')) {
+				observer.observe(node, {childList: true});
+			} else {
+				addRawBtn(node);
+			}
+		}
+	}
+});
+
+function addRawBtn(node: Element | null) {
+	if (node) {
```

Comment:
As in the other PR, node is always Element now

Reply:
Unfortunately, the `select` function returns an `HTMLElement` or null. By passing it as argument of this function when calling it in the `init` method, TS strict mode "forces" me to make this check.

Reply:
You can use `select(selector)!`, the `!` will become your best friend :D 

But https://github.com/sindresorhus/refined-github/commit/a6a6e15d1d0300789b4f548dffb7d5777ad10bc8 avoids this problem too.

Reply:
I didn't know about the `!`. Thanks!